CHANGELOG.md

```diff
@@ -38,10 +39,18 @@ Changed
 - Changed eauth "not enabled" log message level from debug to warning. (#50946)
 -  (#52546)
 - Refactored x509.certificate_managed to be easier to use. (#52935)
+- Don't log error when running "alternatives --display" on nonexistant target (#53911)
 - Improved logging for user auth issues. (#53990)
 - No longer emit extra logs when checking `alternatives.display` and `.check_exists`. (#53991)
 - Use lazy loading to get SLS data from master - significantly improves `state.apply` times when using gitfs with many branches. (#54468)
+- Blacken salt (Implements SEP 15). (#55765)
+- Disable codecov PR comments and status checks (#55888)
 - Changed Salt icon for Windows. (#56194)
+- Replace `log.warn` by `log.warning` and deprecated `ElementTree.getchildren()` (#56226)
+- Docs Replace incorrectly used "it's" by "its" (#56249)
+- Docs: fix typo in windows-package-manager.rst (#56303)
+- Update `libnacl` to 1.7.1 (#56350)
+- Fix docs in win_wua (#56374)
```

Comment:
I'm not sure how user-facing these changes are. Probably the libnacl one is good to keep.